app/src/ui/preferences/integrations.tsx

```diff
@@ -144,43 +136,12 @@ export class Integrations extends React.Component<
     )
   }
 
-  private renderMergeTool() {
-    if (!enableMergeTool()) {
-      return null
-    }
-
-    const mergeTool = this.props.mergeTool
-
-    return (
-      <div className="brutalism">
-        <strong>{__DARWIN__ ? 'Merge Tool' : 'Merge tool'}</strong>
-
-        <Row>
-          <TextBox
-            placeholder="Name"
-            value={mergeTool ? mergeTool.name : ''}
-            onValueChanged={this.props.onMergeToolNameChanged}
-          />
-        </Row>
-
-        <Row>
-          <TextBox
-            placeholder="Command"
-            value={mergeTool && mergeTool.command ? mergeTool.command : ''}
-            onValueChanged={this.props.onMergeToolCommandChanged}
-          />
-        </Row>
-      </div>
-    )
-  }
-
   public render() {
     return (
       <DialogContent>
         <h2>Applications</h2>
         <Row>{this.renderExternalEditor()}</Row>
-        <Row>{this.renderSelectedShell()}</Row>
-        {this.renderMergeTool()}
+        <Row>{this.onSelectedShellChanged()}</Row>
```

Comment:
Shouldn't this be `renderSelectedShell`?

```suggestion
        <Row>{this.renderSelectedShell()}</Row>
```